extensions/amp-base-carousel/1.0/arrow.js

```diff
@@ -15,8 +15,7 @@
  */
 
 import * as Preact from '../../../src/preact';
-import * as styles from './base-carousel.css';
-import {useEffect, useState} from '../../../src/preact';
+import {useStyles} from './base-carousel';
```

Comment:
I'm make a note on this in the `base-carousel.js`, but IMHO this import seems a bit unobvious.

Reply:
moved to .jss.js 👍